### PR TITLE
[OSJC-241] Add int test for authorization kinds

### PR DESCRIPTION
**Pull request by jcantrill**
[test]  cc @adietish @jeffmaury 

Adding to verify #241

### CODE REVIEW

File: src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+ /*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
```
Comment (jeffmaury) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: Be consistent: change to 2016

File: src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	private IClient client;
+	private IntegrationTestHelper helper = new IntegrationTestHelper();
+
+	@Before
+	public void setup () {
+		client = helper.createClientForBasicAuth();
+	}
+	
+	@Test
+	public void testListRolesAssumingClusterAdmin(){
```
Comment (jeffmaury) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: This test keeps failing on both my Win10 and my RHEL 7.2 VM. I'm using CDK 2.3 (OCP 3.3)

Comment (jeffmaury) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: Xunit report for the failed test: https://gist.github.com/jeffmaury/3c46eda295d8377db506e99a9f4db215

Comment (jcantrill) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: Can you confirm:
* You are a cluster admin
* You have roles in the default namespace (using 'oc')

File: src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java (hunk 1 above)
Comment (jcantrill) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: fixed.

File: src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java (hunk 2 above)
Comment (jeffmaury) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: I can't make it to work. Test testListRolesAssumingClusterAdmin is always failing. I used 
`oadm policy add-role-to-user cluster-admin admin`

Comment (jcantrill) [inline src/test/java/com/openshift/internal/restclient/authorization/AuthorizationKindsIntegrationTest.java]: try 
```
oadm policy add-cluster-role-to-user cluster-admin admin
```

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**